.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+jobs:
+   test:
+    docker:
+      - image: ghcr.io/cirruslabs/flutter:3.19.0
+      
+    steps:
+      - checkout
+      - run: flutter doctor
+      - run: flutter pub get
+      - run: flutter pub run build_runner build --delete-conflicting-outputs
+      - run: flutter test
+      - run: flutter analyze .
+      - run: flutter pub publish --dry-run
+      
+   release:
+     docker:
+      - image: ghcr.io/cirruslabs/flutter:latest
+     steps:
+      - checkout
+      - run: ./release.sh
```

Comment:
Do we need to add any other steps like `flutter pub get` before running the `release.sh` script for this job to work?